src/main/resources/application.properties

```diff
@@ -1,5 +1,5 @@
 spring.datasource.username=root
-spring.datasource.password=root
+spring.datasource.password=taur23
```

Comment:
Aici o sa intervina o problema dupa ce mergeuim. Fisierul asta, fiind in git, va trebui sa fie pentru toata lumea la fel, iar daca la tine e diferit si dai push la modificarea asta, atunci toti vor avea parola ta in application.properties-ul lor cand dau pull si vor trebui sa o schimbe manual. Se poate intampla si inversul, adica sa trebuiasca tu sa modifici fisierul asta dde fiecare data cand dai pull. Ar trebui sa-ti schimbi parola ca sa fie la fel ca a celorlalti

Reply:
Aceste date nu trebbuie sa fie puse intr-un git. Let's talk about 'em